### ISSUE TITLE
Update PR Review Wiki with Clarification on Deleting Branch

### ISSUE THREAD

**Issue by kathrynsilvaconway** [hackforla/website#3492]
### Overview
As a developer, I need directions in our PR Review wiki to be as clear as possible. For this issue, we will add a step to the wiki directing devs to delete their branch after reviewing a PR.

### Action Items
- [x] In the wiki called "How To Review a PR Wiki", add the following in step 3, after the sentence "Outlined below are steps to download the collaborator's branch into your own version of the website.", add the following:
```
Before creating a new branch in your repository, always sync your forked repo to the main repo. From your personal  website fork on Github, click "Sync fork" at the right on the screen, then click "Update Branch." You can also update your gh-pages by  running git pull from the command line.
```
 - [x] Add a brief screen capture to demonstrate syncing the fork on Github.
 - [x] At the bottom of the wiki, add a step 7. It should say:
```
In order to keep your working repo as clean as possible, it is best practice to delete your topic branch after you submit your review by running the command: git branch -D name-of-review-branch

This will help to reduce the likelihood of unwanted commits appearing in the future. Always delete the branch as soon as your review is submitted. If you need to go back and re-review the same PR again later, create a new branch at that time.
```
 - [x] Check to make sure that the formatting of the added steps is consistent with the existing steps in this wiki.
 

### Resources/Instructions
[How To Review a PR Wiki](https://github.com/hackforla/website/wiki/How-to-Review-Pull-Requests)
[screen recording app](https://www.cockos.com/licecap/)

**Comment by 7kram** [comment]
@kathrynsilvaconway Hey, we made some formatting edits to this issue to make it more clear, can you confirm the steps are still accurate with your original intention Thanks

**Comment by ExperimentsInHonesty** [comment]
@kathrynsilvaconway please only put the links into the resources section, unless absolutely necessary.  We have added the ready for dev lead label and removed the ready for milestone.  Please put the ready for milestone label back on when you have cleared this issue.  

NOTE to pms: this issue will get a Team Workflow milestone.

**Comment by kathrynsilvaconway** [comment]
@7kram Yes, everything here looks good.

**Comment by bishrfaisal** [comment]
@kathrynsilvaconway Is there a tool that the team commonly uses on Mac and PC for screen recordings? If so, is it in our wiki? Can we link to that wiki page in the resources section?

**Comment by blulady** [comment]
Added screen share software.

**Comment by t-will-gillis** [comment]
i. Availability: 9-5 Thursday 10/26/22
ii. ETA: 5 pm 10/26/22

**Comment by t-will-gillis** [comment]
I have a blocker:

so far:
- I cloned `https://github.com/hackforla/website.wiki.git` into my local branch, 
- made appropriate edits to the file `How-to-Review-Pull-Requests.md`,
- and added a GIF to `image/how-to-review-pull-requests`.
- Also: I am able to preview the changes by copying the edits into the file at `https://github.com/hackforla/website/wiki/How-to-Review-Pull-Requests/_edit`

**Blocker** is with pushing changes to repo. When I run `git status` the subfolder `website.wiki` is misidentified as a file and with `git add .` the following is returned:

![blocker_msg2](https://user-images.githubusercontent.com/40799239/198896487-ab0bb264-aed1-4225-b9a9-e45e746dd2e0.png)


My branch is `update-pr-review-wiki-3492`

I have made several attempts to resolve this but am spinning my wheels. 
Thank you
